src/views/stack_view.rs

```diff
@@ -77,18 +83,22 @@ impl<T: View> ChildWrapper<T> {
     /// Returns a reference to the inner view
     pub fn get_inner(&self) -> &View {
         match *self {
-            ChildWrapper::Shadow(ref shadow) => shadow.get_inner().get_inner(),
-            ChildWrapper::Plain(ref layer) => layer.get_inner(),
+            ChildWrapper::Shadow(ref shadow) => shadow.get_inner(),
+            ChildWrapper::Backfilled(ref background) => background.get_inner(),
+            ChildWrapper::Plain(ref layer) => layer,
         }
     }
 
     /// Returns a mutable reference to the inner view
     pub fn get_inner_mut(&mut self) -> &mut View {
         match *self {
             ChildWrapper::Shadow(ref mut shadow) => {
-                shadow.get_inner_mut().get_inner_mut()
+                shadow.get_inner_mut()
```

Comment:
Same thing here, any reason you removed a round of `get_inner_mut()`?

Reply:
Those are my fault, it seems I haven't properly restored everything after the previous attempt.